src/lib/plugin.js

```diff
@@ -25,6 +25,7 @@ const defaultConnectTimeout = 60000
 const wsReconnectDelayMin = 10
 const wsReconnectDelayMax = 500
 const defaultMessageTimeout = 5000
+const authTokenMaxAge = 7 * 24 * 60 * 60 * 1000 // one week
```

Comment:
Will the websocket stop working if the token becomes too old? If so, we might want to set this age to less than a week, so that it's refreshed before it expires.